src/hooks/useModelStatus.ts

```diff
@@ -20,15 +19,9 @@ export default function useModelStatus() {
     modelName = queryParams[0].model;
   }
 
-  const getModelUUIDMemo = useMemo(
-    () => (modelName ? getModelUUID(modelName) : null),
-    [modelName],
-  );
-  const modelUUID = useAppSelector((state) => getModelUUIDMemo?.(state));
-  const getModelStatusMemo = useMemo(
-    () => getModelStatus(modelUUID),
-    [modelUUID],
+  const modelUUID = useAppSelector((state) =>
+    modelName ? getModelUUID(state, modelName) : null,
```

Comment:
Instead of conditionally calling the selector it might be easier to update `getModelUUID` to take a null/undefined modelName and exit early if the name is not passed.